syntax = "proto3";
import "Common_Base.proto";
import "DB_Base.proto";
package  PManager_Client;  
/*
协议规则前两个字母代表消息流向
D:数据服务器
M：管理服务器
L：登录服务器
G：逻辑服务器
C：客户端
后面更协议的名字
命令枚举以服务器前客户端后的命名_CMD,一对协议对应一个消息id，推送协议占一个消息id

管理服-客户端消息：6001-8000
*/

enum EMC_CMD
{
    EMC_CMD_None=0;
	EMC_CMD_Login =1;
}

message CM_Login 
{  
	int64 Uid=1;
	string key=2;
}  

message MC_Login 
{  
	int32 ret=1;
	string GameIp=2;
    int32 GamePort=3;
    string key=4;
}  
